Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -3524,6 +3556,24 @@ public static void OpenALootBoxForAssetInstances(int assetInstanceID, Action<Loo
             data.getRequests.Add(assetInstanceID.ToString());
             LootLockerAPIManager.OpenALootBox(data, onComplete);
         }
+
+        /// <summary>
+        /// Delete an Asset Instance from the current Player's Inventory.
+        /// </summary>
+        /// <param name="assetInstanceID">The asset instance ID of the you want to delete from the Players Inventory</param>
```

Comment:
Missing some words, suggesting to write;
"The asset instance ID of the asset instance you want to delete from the current Players Inventory"